### ISSUE TITLE
error on download

### ISSUE THREAD

**Issue by dominikkukacka** [pelias/openaddresses#334]
Hi,

somehow I am getting this error on download
```
user@server:/opt/pelias/openaddresses# node utils/download_data.js
2018-01-24T19:27:11.348Z - info: [download] Attempting to download all data
2018-01-24T19:27:11.353Z - debug: [download] downloading https://s3.amazonaws.com/data.openaddresses.io/openaddr-collected-global.zip
2018-01-24T19:52:11.771Z - debug: [download] unzipping /tmp/tmp-10khLEclwr1Pwr.zip to /data/openaddresses
2018-01-24T19:56:48.095Z - error: [download] Failed to download data Error: Command failed: unzip -o -qq -d /data/openaddresses /tmp/tmp-10khLEclwr1Pwr.zip
```

unzip is installed:
```
user@server:/opt/pelias/openaddresses# unzip
UnZip 6.00 of 20 April 2009, by Debian. Original by Info-ZIP.

Usage: unzip [-Z] [-opts[modifiers]] file[.zip] [list] [-x xlist] [-d exdir]
...
```

Any ideas?

**Comment by orangejulius** [comment]
Hey @dominikkukacka,
Hmm. Well we definitely need to improve the error message.

I get something similar, although I don't get an error. Instead `unzip` just hangs. Maybe it's due to the increasingly large size of the OA download file (it's now up to 9.5GB).

**Comment by dominikkukacka** [comment]
Hey @orangejulius,
I ran the downloader twice again. The last time without any other downloader/importer running. And then it worked. The unzip took ages but it did work.

Short question: How many entries should a planet build contain? SOmehow after running all the importers I only got 331M entries. It should be 509M no?

![image](https://user-images.githubusercontent.com/243319/35497157-4cd6b86c-04c9-11e8-8542-b45eda53208f.png)

Any idea what is missing?
